client/src/components/Account/ResetPassword.js

```diff
@@ -38,7 +38,10 @@ const styles = (theme) => ({
 const validationSchema = Yup.object().shape({
   token: Yup.string().required("Token is required"),
   password: Yup.string()
-    .min(8, "Password must be 8 characters at minimum")
+    .matches(
+      /^.*(?=.{8,})((?=.*[!@#$%^&*()\-_=+{};:,<.>]){1})(?=.*\d)((?=.*[a-z]){1})((?=.*[A-Z]){1}).*$/,
```

Comment:
can we move the regex on a helper file so we can just reuse it?😅

Reply:
Stored it in the `Constants.js` file